### PR TITLE
docs: add mkdocs+pdoc setup

### PR DESCRIPTION
**Pull request by jstriebel**
### Description:
Changes the main documentation to use mkdocs (with material theme), but still integrates the pdoc documentation. Some initial pages are added, but mostly as an infrastructure for further improvement.

Please see `docs/README.md` for usage details.

The published version of this branch is available at
https://static.webknossos.org/lib-docs/mkdocs/index.html

Further steps might include (tracked in the issue):
* Show examples prominently
* Specific guides and tutorials
* Version selection (see https://squidfunk.github.io/mkdocs-material/setup/setting-up-versioning & https://github.com/squidfunk/mkdocs-material-example-versioning/tree/gh-pages)
* Auto-forward `/` to `/index.html`

### Issues:
- part of #379

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
-## [webKnossos cuber (wkcuber)](wkcuber)
-[![PyPI version](https://img.shields.io/pypi/v/wkcuber)](https://pypi.python.org/pypi/wkcuber) [![Supported Python Versions](https://img.shields.io/pypi/pyversions/wkcuber.svg)](https://pypi.python.org/pypi/wkcuber)
+Currently under construction :construction:
 
-Python library for creating and working with [webKnossos](https://webknossos.org) [WKW](https://github.com/scalableminds/webknossos-wrap) datasets. WKW is a container format for efficiently storing large, scale 3D image data as found in (electron) microscopy.
-
-## [webKnossos](webknossos) Python Library
-:construction: WIP
+## [webKnossos cuber (wkcuber) CLI](wkcuber)
+CLI tool to creating and manipulate [webKnossos](https://webknossos.org) [WKW](https://github.com/scalableminds/webknossos-wrap) datasets. WKW is a container format for efficiently storing large, scale 3D image data as found in (electron) microscopy.
```
Comment (normanrz) [inline README.md:14]:
```suggestion
CLI tool to creating and manipulate [webKnossos](https://webknossos.org) [WKW](https://github.com/scalableminds/webknossos-wrap) datasets. WKW is a container format for efficiently storing large-scale 3D images as found in microscopy data.
```

**Review by normanrz — APPROVED** [review]
looks nice!

**Comment by philippotto** [comment]
Cool stuff! There are some dead links when clicking through the pages, though. Namely:

- https://static.webknossos.org/lib-docs/mkdocs/
- https://static.webknossos.org/lib-docs/mkdocs/webknossos
- https://static.webknossos.org/lib-docs/mkdocs/wkcuber

I think, the missing `index.html` is throwing things off.

**Comment by jstriebel** [comment]
> I think, the missing index.html is throwing things off.

Yes, but not the `index.html` is missing but the forward to it from S3. I noted this in the next steps (also in the issue).

**Comment by jstriebel** [comment]
PS: @normanrz You could check that in the S3 settings? If that's complicated, we could consider moving to gh-pages.

**Pull request merged**